firewood/tests/db.rs

```diff
@@ -100,10 +89,11 @@ fn test_revisions() {
         key
     };
     for i in 0..10 {
-        let db =
-            PersistedDb::new("test_revisions_db", &cfg.clone().truncate(true).build()).unwrap();
+        let db = PersistedDb::new("test_revisions_db", &cfg.clone().truncate(true).build())
+            .await
+            .unwrap();
         let mut dumped = VecDeque::new();
-        let mut hashes: VecDeque<TrieHash> = VecDeque::new();
+        let mut hashes: VecDeque<firewood::v2::api::HashKey> = VecDeque::new();
```

Comment:
Nit: prefer qualified path use.
```suggestion
        let mut hashes: VecDeque<HashKey> = VecDeque::new();
```